content/concepts/tokens/nftoken-auctions.md

```diff
@@ -0,0 +1,64 @@
+---
+html: nftoken-auctions.html
+parent: non-fungible-tokens.html
+blurb: You can assign another account to mint NFTs in your stead.
+labels:
+ - Non-fungible Tokens, NFTs
+---
+
+# Running an NFT Auction
+
+There are several ways to run an auction, each with advantages and disadvantages.
+
+## Run the Auction Off the XRPL, Complete the Purchase on XRPL
+
+This flow is the most straightforward. Note that the `NFTokenOffer` objects can always be canceled by their creator, so it's not possible to implement a binding offer.
+
+1. Store your bids in a private database.
+2. You take a cut of the winning bid.
+3. Send the buyer/seller the XRPL transaction to complete the purchase. 
+
+## Run the Auction in Brokered Mode, with a Reserve
+
+Run the auction in brokered mode, as an auction with a reserve.
+
+![Auction in Brokered Mode with a Reserve](img/nft-auction1.png "Auction in Brokered Mode with a Reserve")
+
+1. The Seller creates the NFT, then sets the auction reserve price using `NFTokenCreateOffer`, specifying the broker account as the destination.
+1. Buyers make bids using `NFTokenCreateOffer`, setting the broker account as the destination.
+1. The Broker selects the winning bid, completes the sale using `NFTokenAcceptOffer`, collecting the broker fee. Then the Broker cancels any losing bids using `NFTokenCancelOffer`.
+
+**Pros:**
+
+- The entire auction happens on the XRPL, including your broker fee.
+- The seller represents their reserve price on-chain.
+- This is _close_ to a binding offer, from the buyside.
+
+**Cons:**
+
+- There must be implicit trust between the seller and the marketplace that the marketplace will not take more than some previously-agreed-upon rate. If the reserve was 1 XRP and the winning bid was 1000 XRP, there is no on-chain mechanism to prevent the broker from taking 999 XRP as profit, leaving only the reserve profits for the Seller.
```

Comment:
nit: I'm noticing some inconsistencies on the capitalization for "Buyer", "Seller", and "Broker", do we intentionally want to have different variants based on sentence context?

Reply:
maybe its good to clarify that marketplace = broker?